gperror/gperror_test.go

```diff
@@ -0,0 +1,63 @@
+package gperror_test
+
+import (
+	"errors"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/greenplum-db/gp-common-go-libs/gperror"
+)
+
+func TestGpError(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Gperror Suite")
+}
+
+var _ = Describe("gperror", func() {
+	var testErr *gperror.GpError
+
+	BeforeEach(func() {
+		testErr = &gperror.GpError{
+			ErrorCode: gperror.ErrorCode(4321),
+			Err:       errors.New("test-error"),
+		}
+	})
+
+	Context("Error", func() {
+		When("the function is called", func() {
```

Comment:
Stylistically, the tests for the other packages use `Describe` instead of `Context`, and don't use `When` because it doesn't generally add any useful information for simple unit tests.  If you can tweak these tests to match, that would be great.

Everything looks good otherwise.

Reply:
Addressed in most recent commit.